dashboard/app/controllers/experiments_controller.rb

```diff
@@ -22,7 +22,7 @@ def set_course_experiment
     redirect_to '/', flash: {notice: "You have successfully joined the experiment '#{params[:experiment_name]}'."}
   end
 
-  VALID_EXPERIMENTS = ['2018-teacher-experience'] + Experiment::PILOT_EXPERIMENTS.collect {|p| p[:name] if p[:allow_joining_via_url]}.compact
+  VALID_EXPERIMENTS = ['2018-teacher-experience', 'lesson-group'] + Experiment::PILOT_EXPERIMENTS.collect {|p| p[:name] if p[:allow_joining_via_url]}.compact
```

Comment:
no need to add this here, unless you are planning to send users to `/experiments/set_single_user_experiment/:experiment_name` to enable it